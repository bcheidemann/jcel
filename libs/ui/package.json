{
  "name": "@jcel/ui",
  "version": "0.0.1"
}
