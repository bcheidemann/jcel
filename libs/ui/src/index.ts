export * from './lib/editor/editor';
export * from './lib/sheet/sheet';
