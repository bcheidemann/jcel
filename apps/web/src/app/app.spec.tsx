import React from 'react';
import { render } from '@testing-library/react';
import { Editor, Sheet } from '@jcel/ui';
import { mocked } from 'ts-jest';

import App from './app';

jest.mock('@jcel/ui', () => ({
  Editor: () => '[EDITOR]',
  Sheet: () => '[SHEET]',
}));

describe('App', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<App />);

    expect(baseElement).toBeTruthy();
  });

  it('should match the snapshot', () => {
    const { baseElement } = render(<App />);

    expect(baseElement).toMatchSnapshot();
  });
});
