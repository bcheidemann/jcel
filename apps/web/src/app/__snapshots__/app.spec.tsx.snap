// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<body>
  <div>
    <div
      style="position: absolute; top: 0px; bottom: 0px; left: 0px; right: 0px; display: flex;"
    >
      [SHEET]
      <div
        style="width: 500px; background-color: white; display: flex; flex-direction: column;"
      >
        [EDITOR]
        <button>
          Save
        </button>
      </div>
    </div>
  </div>
</body>
`;
